build.sbt

```diff
@@ -53,9 +53,9 @@ lazy val `aws-parameterstore-lambda` = project.in(file("aws-parameterstore/lambd
 lazy val `secret-generator` = project.settings(crossCompileScala3, baseSettings)
 
 val exactPlayVersions = Map(
-  "27" -> "com.typesafe.play" %% "play" % "2.7.9",
-  "28" -> "com.typesafe.play" %% "play" % "2.8.20",
-  "29" -> "com.typesafe.play" %% "play" % "2.9.0",
+  "27" -> "org.playframework" %% "play" % "2.7.9",
+  "28" -> "org.playframework" %% "play" % "2.8.21",
```

Comment:
The new `org.playframework` group id above here is incorrect, as that group id only came into effect with Play 3 - see https://github.com/scala-steward-org/scala-steward/pull/3191/files#r1395531926